app/views/songs/index.html.erb

```diff
@@ -39,7 +39,9 @@
       serverSide: true,
       responsive: true,
       lengthChange: false,
-      dom: '',
+      pageLength: 25,
```

Comment:
Once we deploy to prod with all the new songs, if it's slow, we can try upgrading the app and even the DB and decide if we want to delay the query. Thanks for your hard work!